timescaledb/how-to-guides/hypertables/hypertables-and-unique-indexes.md

```diff
@@ -0,0 +1,140 @@
+# Hypertable partitioning and unique indexes
+Hypertable partitioning introduces some restrictions on unique indexes.
+
+## Partitioning columns
+To determine what unique indexes you can create on a hypertable, first
+identify your partitioning columns.
+
+In TimescaleDB, the following columns are partitioning columns:
```

Comment:
```suggestion
In TimescaleDB, you can use these columns for partitioning:
```

Reply:
This is less clear, these are the partitioning columns of a hypertable, there's no option, we use them for partitioning under the hood.

Reply:
```suggestion
TimescaleDB uses these columns for partitioning hypertables:
```

Reply:
That's fine with me.